tests/unit/single_column/statistical/test_boundary_adherence.py

```diff
@@ -31,6 +33,65 @@ def test_compute(self):
         # Assert
         assert result == 0.75
 
+    def test_compute_nans(self):
+        """Test the ``compute`` method with nan values.
+
+        Expect that the nan values in synthetic data are considered as
+        out of bounds if the real data does not also containt nan values.
+        """
+        # Setup
+        real_data = pd.Series([1.0, 2.4, 2.6, 0.8])  # 0.8 -> 2.6
+        real_data_nans = pd.Series([1.0, 2.4, 2.6, 0.8, np.nan])
+        synthetic_data = pd.Series([0.9, 1.8, 2.1, 5.0, np.nan])
+
+        metric = BoundaryAdherence()
+
+        # Run
+        result = metric.compute(real_data, synthetic_data)
+        result_ignore_nans = metric.compute(real_data_nans, synthetic_data)
+
+        # Assert
+        assert result == 0.6
+        assert result_ignore_nans == 0.75
+
+    def test_compute_datetime_nans(self):
+        """Test the ``compute`` method with nan values.
+
+        Expect that the nan values in synthetic data are considered as
+        out of bounds if the real data does not also containt nan values.
+        """
+        # Setup
+        real_data = pd.Series([
+            datetime(2020, 10, 1),
+            datetime(2021, 1, 2),
+            datetime(2021, 9, 12),
+            datetime(2022, 10, 1),
+
+        ], dtype='datetime64[ns]')  # 0.8 -> 2.6
```

Comment:
What does indicate # 0.8 -> 2.6 in the datetime case?

Reply:
Ah good catch, that was just to help me write the test 😅